examples/bundle.js

```diff
@@ -5355,16 +5356,15 @@
                 this.isValid = isValid;
                 this._errors = errors;
             };
-        }
-        connectedCallback() {
-            var _a, _b;
             const { onSubmit, initialValues, validationSchema, validateOnChange, validateOnBlur } = config !== null && config !== void 0 ? config : {};
             // take params from HOC argument, or from class (if you build base class, like <lite-form>) or default
             this._onSubmit = (onSubmit || this.onSubmit || function () { }).bind(this);
             this._initialValues = initialValues || this.initialValues || {};
             this._validationSchema = validationSchema || this.validationSchema || {};
             this._validateOnBlur = (_a = validateOnBlur !== null && validateOnBlur !== void 0 ? validateOnBlur : this.validateOnBlur) !== null && _a !== void 0 ? _a : true;
```

Comment:
is `_a` and `_b` still in scope and available here if you transfer their initialisation in constructor?

Reply:
This is the generated bundle. I expect it to be correct if the source is correct so I haven't checked it.

The reason why I moved init to the constructor is that I changed the _props to readonly and those could only be initialized in the constructor.